src/librustdoc/html/render/mod.rs

```diff
@@ -4504,7 +4504,7 @@ fn extract_for_impl_name(item: &clean::Item) -> Option<(String, String)> {
 }
 
 fn is_negative_impl(i: &clean::Impl) -> bool {
-    i.polarity == Some(clean::ImplPolarity::Negative)
+    i.negative_polarity
 }
```

Comment:
I would rather get rid of this function altogether I think, and just check this field directly.

Reply:
Fine by me!